File: java/src/test/java/app/endpoints/WebEndpoints.java
package app.endpoints;

import static app.config.Utils.*;
import static io.gatling.javaapi.http.HttpDsl.*;

import io.gatling.javaapi.http.HttpRequestActionBuilder;

public class WebEndpoints {

  public static final HttpRequestActionBuilder homePage =
      http("HomePage").get(pageUrl).check(status().in(200, 304));

  public static final HttpRequestActionBuilder loginPage =
      http("LoginPage").get(pageUrl + "/login").check(status().in(200, 304));
  ;
}

File: scala/src/test/scala/app/endpoints/WebEndpoints.scala
package app.endpoints

import io.gatling.core.Predef._
import io.gatling.http.Predef._
import app.config.Utils._

object WebEndpoints {

  val homePage = http("HomePage")
    .get(pageUrl)
    .check(status.in(200, 304))

  val loginPage = http("LoginPage")
    .get(s"$pageUrl/login")
    .check(status.in(200, 304))
}
